### ISSUE TITLE
Live demo link should point to English version

### ISSUE THREAD

**Issue by emiryuksel** [emiryuksel/LiteBot#2]
The current live demo link in the README points to the Turkish version of the site. To ensure accessibility for a broader audience, it should be updated to the English version.

Current link: https://emiryuksel.site/projects/litebot
Correct link: https://emiryuksel.site/en/projects/litebot

Will update the README accordingly.
